src/common/options/osd.yaml.in

```diff
@@ -326,7 +326,7 @@ options:
   type: float
   level: dev
   desc: Backoff ratio for scheduling scrubs
-  long_desc: This is the precentage of ticks that do NOT schedule scrubs, 66% means
+  long_desc: This is the percentage of ticks that do NOT schedule scrubs, 66% means
```

Comment:
It's a per-tick() coin-flip (OSD::scrub_random_backoff(), right?).  How about:

"Probability that a particular OSD tick instance will skip scrub scheduling.  66% means that approximately 1/3 ticks will cause scrub scheduling"

Longer term, I'm not actually sure there's a benefit to this config.  It's not a "backoff" in the traditional sense of a randomized interval before retrying after a failure.  If the scrub scheduling action in tick() is simply checking a data structure optimized for the purpose, it seems fine to just do it every time and would allow us to drop a config.

Reply:
Agree.